tools/python-integration-tests/deployment.py

```diff
@@ -26,7 +26,7 @@ def __init__(self, blueprint: str):
         self.workspace = None
         self.instance_name = None
         self.username = None
-        self.deployment_name = None
+        self.deployment_name = os.environ.get('BUILD_ID_SHORT') 
```

Comment:
We need to account for non-CloudBuild invocation. Let's default to short uuid if `BUILD_ID` is not present.

Reply:
Done